libconcorde_tsp_solver/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<package>
+	<name>libconcorde_tsp_solver</name>
+	<version>0.1.0</version>
+	<description>Wrapper for the concorde traveling salesman problem solver from http://www.math.uwaterloo.ca/tsp/concorde/downloads/downloads.htm</description>
+
+	<author email="florian.jordan@ipa.fraunhofer.de">Florian Jordan</author>
+	<maintainer email="rmb@ipa.fhg.de">Richard Bormann</maintainer>
+
+	<license>see original software</license>
```

Comment:
what is the license of the original software? 

anyway, the license we list here is just for our ROS wrapper. We should use LGPL here.